tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -401,7 +401,8 @@ def test_fsdp_mixed_with_sync(
         '0.17.0',
         marks=pytest.mark.filterwarnings((r'ignore:MosaicMLLogger is not in the state_dict. Its '
                                           r'state will not be restored.:UserWarning')),
-    )
+    ),
+    pytest.param('0.18.1'),
```

Comment:
```suggestion
    '0.18.1',
```
param not needed if no marks